applications/prompt-proto-service-latiss/values-usdfdev-prompt-processing.yaml

```diff
@@ -13,7 +13,6 @@ prompt-proto-service:
   instrument:
     pipelines: >-
       (survey="SURVEY")=[${PROMPT_PROCESSING_DIR}/pipelines/LATISS/ApPipe.yaml,
-      ${PROMPT_PROCESSING_DIR}/pipelines/LATISS/SingleFrame.yaml,
```

Comment:
I'm not sure we will ever want to fall back on pipeline errors (especially if the "fallback" just means rerunning something that already ran!), but 👍 for something that's easy to roll back later.